### PR TITLE
DEX-562 Rework error codes

### PR DESCRIPTION
**Pull request by alexander-branevskiy**

### CODE REVIEW

File: dex/src/test/scala/com/wavesplatform/dex/api/ws/actors/WsExternalClientHandlerActorSpec.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ import com.wavesplatform.dex.domain.account.KeyPair
 import com.wavesplatform.dex.domain.asset.Asset.{IssuedAsset, Waves}
 import com.wavesplatform.dex.domain.asset.AssetPair
 import com.wavesplatform.dex.domain.bytes.ByteStr
-import com.wavesplatform.dex.error.SubscriptionsLimitReached
+import com.wavesplatform.dex.error.{AddressAndPublicKeyAreIncompatible, InvalidJwtPayloadSignature, JwtBroken, JwtCommonError, JwtPayloadBroken, OrderAssetPairReversed, RequestArgumentInvalid, SubscriptionAuthTypeUnsupported, SubscriptionTokenExpired, SubscriptionsLimitReached, TokenNetworkUnexpected}
```
Comment (JennaWestenra) [inline dex/src/test/scala/com/wavesplatform/dex/api/ws/actors/WsExternalClientHandlerActorSpec.scala:23]: Maybe, replace it with `import com.wavesplatform.dex.error._` ?

Comment (ngadiyak) [reply dex/src/test/scala/com/wavesplatform/dex/api/ws/actors/WsExternalClientHandlerActorSpec.scala:23]: +1

**Review by ngadiyak — APPROVED** [review]

File: dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/RawHttpChecks.scala [hunk 2]
```diff
@@ -39,10 +40,10 @@ trait RawHttpChecks extends Matchers {
     r.response.code should be(StatusCode.MovedPermanently)
 
   protected def validateIncorrectSignature[ErrorT, EntityT](r: EnrichedResponse[ErrorT, EntityT]) =
-    validateMatcherError(r, StatusCode.BadRequest, 1051904, "The request has an invalid signature")
+    validateMatcherError(r, StatusCode.BadRequest, RequestInvalidSignature.code, "The request has an invalid signature")
```
Comment (vsuharnikov) [inline dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/RawHttpChecks.scala:43]: 👍

**Review by vsuharnikov — APPROVED** [review]

File: dex/src/test/scala/com/wavesplatform/dex/api/ws/actors/WsExternalClientHandlerActorSpec.scala (hunk 1 above)
Comment (alexander-branevskiy) [reply dex/src/test/scala/com/wavesplatform/dex/api/ws/actors/WsExternalClientHandlerActorSpec.scala]: resolved

**Review by JennaWestenra — APPROVED** [review]

**Pull request merged**